gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=544c35d6bd849ae8a5ed0bcea39ba677dc40f49df7d1835561582da2009b961d
```

Comment:
💅 I think it would be nice to preserve the sha sum

Reply:
Yes, will do that once there is a released version to use.